src/frontend/komponenter/Fagsak/Behandlingsresultat/EøsPeriode/FamilieLandvelger.tsx

```diff
@@ -247,7 +247,7 @@ const FamilieMultiLandvelger: React.FC<IFamilieMultiLandvelgerProps> = ({
     );
 };
 
-interface IFamilieValutavelgerProps extends IBaseFamilieLandvelgerProps {
+export interface IFamilieValutavelgerProps extends IBaseFamilieLandvelgerProps {
```

Comment:
Dette gjør det mulig å lage en `styled(FamilieLandvelger)` i UtenlandskPeriodeBeløpTabellRadEndre` som igjen eksporteres og som typescript forstår proptypingen for.